docs/develop/01_sdk/01_quickstart.md

```diff
@@ -91,15 +90,6 @@ For this, the SDK exposes **`Kilt.init()`** to configure the address of the node
 
 We will use the official **Spiritnet** address:
 
-<SnippetBlock
-  className="language-ts"
->
-  {InitSDK}
-</SnippetBlock>
-
-Once the node address has been configured, we can establish a connection with the Spiritnet node.
-Again, the SDK makes this process very easy to follow, by exposing a `Kilt.connect()` function:
```

Comment:
We have a mismatch here now with the text mentioning `init()` and the snippet using `connect()`

Reply:
Good catch! I fixed that.